package/MDAnalysis/analysis/waterdynamics.py

```diff
@@ -639,7 +639,10 @@ def _getMeanOnePoint(self,universe,selection1,selection_str,dt,totalFrames):
         valdipList = []
 
         for j in range(totalFrames/dt-1):
-            a = self._getOneDeltaPoint(universe,repInd,j,sumsdt,dt)
+            try:
+                a = self._getOneDeltaPoint(universe,repInd,j,sumsdt,dt)
+            except ZeroDivisionError:
+                continue
```

Comment:
Is it really ok to just ignore the division by zero? Or should this be fixed elsewhere?